src/test/java/bio/terra/service/snapshotbuilder/SnapshotRequestDaoTest.java

```diff
@@ -129,11 +158,36 @@ void createDatasetIdNotFound() {
         () -> snapshotRequestDao.create(UUID.randomUUID(), snapshotAccessRequest, EMAIL));
   }
 
+  @Test
+  void createV2() {
+    SnapshotAccessRequestResponse response = createRequestV2();
+    verifyResponseContents(response, true);
+
+    SnapshotAccessRequestResponse response1 =
+        snapshotRequestDao.createV2(snapshot.getId(), snapshotAccessRequest, EMAIL);
+
+    Assertions.assertNotEquals(
+        response1.getId(),
+        response.getId(),
+        "Snapshot Access Request Response should have unique request id");
```

Comment:
You can statically import `assertNotEquals`, or use `assertThat(x, not(y))` instead.
```suggestion
    assertThat(
        "Snapshot Access Request Response should have unique request id",
        response1.getId(),
        not(response.getId()));
```